### ISSUE TITLE
On invoking the 'Replace All' button, NVDA/JAWS are not narrating any information while navigating using down arrow.

### ISSUE THREAD

**Issue by lopapraharaj** [tinymce/tinymce#7171]
**What is the current behavior? Describe the bug**

On invoking the 'Replace All' button in HTML editor, screen 
<img width="1614" alt="Screen Shot 2021-09-02 at 1 01 53 PM" src="https://user-images.githubusercontent.com/88460804/131908581-5fb0cc09-79bf-4296-97b9-4a63c53316aa.png">
readers such as NVDA/JAWS are not narrating any information while navigating using down arrow.

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**

**What is the expected behavior?**

NVDA/JAWS should narrate Replaced texts information On invoking the 'Replace All' button while navigating using down arrow. NVDA/JAWS should announce number of replacements done

Ex: NVDA/JAWS should narrate as like 'All done we made 3 replacements'.

WCAG Reference:

https://w3c.github.io/wcag21/understanding/status-messages.html

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**

This behavior is consistent in both their version 4 and version 5
